### PR TITLE
add popularity baseline comparisons

### PR DESCRIPTION
**Pull request by yifeim**
*Description of changes:*
Add popularity baselines as examples for external evaluation methods.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: personalize_temporal_holdout/personalize_temporal_holdout.ipynb [hunk 1]
```diff
@@ -1189,6 +1173,246 @@
...
+    "print('precision_at_5', np.mean([precision_at_k(r, 5) for r in rel_pop_dedup]))\n",
+    "print('precision_at_10', np.mean([precision_at_k(r, 10) for r in rel_pop_dedup]))\n",
+    "print('precision_at_25', np.mean([precision_at_k(r, 25) for r in rel_pop_dedup]))\n",
+    "print('normalized_discounted_cumulative_gain_at_5', np.mean([ndcg_at_k(r, 5) for r in rel_pop_dedup]))\n",
+    "print('normalized_discounted_cumulative_gain_at_10', np.mean([ndcg_at_k(r, 10) for r in rel_pop_dedup]))\n",
+    "print('normalized_discounted_cumulative_gain_at_25', np.mean([ndcg_at_k(r, 25) for r in rel_pop_dedup]))"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (hyunjoons) [inline personalize_temporal_holdout/personalize_temporal_holdout.ipynb]: Can you remove this empty cell?

File: personalize_temporal_holdout/personalize_temporal_holdout.ipynb [hunk 2]
```diff
@@ -1189,6 +1173,246 @@
     "print('normalized_discounted_cumulative_gain_at_10', np.mean([ndcg_at_k(r, 10) for r in relevance]))\n",
     "print('normalized_discounted_cumulative_gain_at_25', np.mean([ndcg_at_k(r, 25) for r in relevance]))"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### slightly better results after deduplicating previous purchase histories"
```
Comment (hyunjoons) [inline personalize_temporal_holdout/personalize_temporal_holdout.ipynb]: Can you say "Observe slightly better results after deduplicating previous purchase histories" or something similar with a capitalized first word?

Comment (yifeim) [reply personalize_temporal_holdout/personalize_temporal_holdout.ipynb]: How about:
### Optional: observe slightly better results after deduplicating previous purchase histories

Comment (yifeim) [reply personalize_temporal_holdout/personalize_temporal_holdout.ipynb]: Done.

**Comment by hyunjoons** [comment]
Thanks, Yifei.

**Pull request merged**